package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@getsetup-io/gsu-embedded-shell",
+  "name": "@getsetup/gsu-embedded-shell",
```

Comment:
Updated the org name to remove the `-io` but leaving the old package name for now, until we sort out the publishing workflow we want.